### DISCUSSION TITLE
SEMP-1.3.xsd

### DISCUSSION THREAD

**Discussion by littleyoda** [camueller/SmartApplianceEnabler#614] [Allgemein]
Hi

wo stammt eigentlich die [SEMP1-3.xsd](https://github.com/camueller/SmartApplianceEnabler/blob/master/xsd/SEMP-1.3.xsd) Datei her?
In den verfügbaren SMA Dokumentationen finde ich immer nur die SEMP-1.1.5.

Danke

**Comment by camueller** [comment]
Sieht man auch in der git-History: Sie wurde mir von SMA zur Verfügung gestellt.

**Comment by littleyoda** [comment]
Soweit habe ich nicht gedacht. Dank dir.
